app/controllers/simplified-account/settings/organisation-details/edit-organisation-details.controller.js

```diff
@@ -21,10 +25,34 @@ function get (req, res) {
 }
 
 function post (req, res) {
+  const errors = validationResult(req)
```

Comment:
How does this `validationResult` work with the `postValidation`, is it magic?

Reply:
Exporting them together as an array of `[postValidation, post]` then applying that to the route makes the validation method act as a middleware, so any errors can attached to the request and it's then passed on to the `post` method